oojs.js

```diff
@@ -1,12 +1,21 @@
-// Define a Playlist class. It should have...
-  // A title property that is determined by some input.
-  // A songs property that is determined by some input. It should contain multiple songs, each of which can just be a song title.
-  // A favorites property that is initialized at 0. This cannot be set by user input.
-  // An addSong method that adds a song to the songs property.
+class Playlist {
+  constructor(title, song1, song2) {
+  this.title = title;
+  this.songs = [song1, song2];
```

Comment:
This works, although it'd probably be better to pass in a `songs` argument that takes an array of songs. That way you're not limiting yourself to only two songs for each playlist.